o8g/Extractor/config.xml

```diff
@@ -0,0 +1,76 @@
+﻿<xml>
+  <game cardsUrl="https://api.fiveringsdb.com/cards"
+        packsUrl="https://api.fiveringsdb.com/packs"
+        gameGuid="bb0f02e7-2a6f-4ae3-84a2-c501b4176844"
+        packId="id"
+        cardNumber="position"
+        cardOctgnNumber="Card Number"
+        cardName="name"
+        cardPackId="pack_code"
+        cardId="octgn_id"
+        cardImage="image_url"
+        />
+  <images>
+    <image name="FiveringsDB" url="{0}" />
+  </images>
+  <properties>
+    <property octgn_name="Clan" type="property" value="clan" capitalize="True" />
+    <property octgn_name="Text" type="property" value="text" isRich="True"/>
+    <property octgn_name="Type" type="property" value="type" capitalize="True" />
+    <property octgn_name="Traits" type="property" value="traits" capitalize="True" delimiter=". " />
+    <property octgn_name="Ring" type="property" value="element" capitalize="True" />
+    <property octgn_name="Card Number" type="property" value="position" />
+    <property octgn_name="Cost" type="property" value="cost" />
+    <property octgn_name="Military Skill" type="property" value="military" />
+    <property octgn_name="Military Skill" type="property" value="military_bonus" />
+    <property octgn_name="Political Skill" type="property" value="political" />
+    <property octgn_name="Political Skill" type="property" value="political_skill" />
+    <property octgn_name="Bonus Strength" type="property" value="strength" />
+    <property octgn_name="Bonus Strength" type="property" value="strength_bonus" />
+    <property octgn_name="Glory" type="property" value="glory" />
+    <property octgn_name="Deck" type="property" value="side" capitalize="True" />
+    <property octgn_name="Unique" type="property" value="unicity" >
+      <replace match="true" value="Unique" />
+      <replace match="false" value="" />
+    </property>
+    <property octgn_name="Influence Value" type="property" value="influence_cost" />
+    <property octgn_name="Influence Value" type="property" value="influence_pool" />
+    <property octgn_name="Starting Honor" type="property" value="honor" />
+    <property octgn_name="Fate Value" type="property" value="fate" />
+  </properties>
+  <sizes>
+    <size name="dynasty">
+      <match property="Deck" value="Dynasty"/>
+    </size>
+    <size name="conflict">
+      <match property="Deck" value="Conflict"/>
+    </size>
+    <size name="province">
+      <match property="Deck" value="Province"/>
+    </size>
+    <size name="province">
```

Comment:
Role and Stronghold didn't have a size previously

Reply:
Strongholds have the same card back as provinces, so it made sense to add it there.  I assumed that role cards are gonna have that back as well, I guess we won’t know for sure until the phoenix pack comes out.

Reply:
Ok yah that makes sense